data/mods/Magiclysm/Spells/magus.json

```diff
@@ -63,5 +63,110 @@
     "base_casting_time": 600,
     "energy_source": "MANA",
     "base_energy_cost": 350
+  },
+  {
+    "id": "magus_mana_blast",
+    "type": "SPELL",
+    "name": "Mana Blast",
+    "description": "A blast of concentrated magical power that obliterates a large area.",
+    "valid_targets": [ "hostile", "ground" ],
+    "effect": "projectile_attack",
+    "spell_class": "MAGUS",
+    "energy_source": "MANA",
+    "base_casting_time": 150,
+    "base_energy_cost": 200,
+    "difficulty": 5,
+    "max_level": 20,
+    "min_damage": 20,
+    "max_damage": 80,
+    "damage_increment": 3,
+    "damage_type": "none",
+    "min_aoe": 1,
+    "max_aoe": 3,
+    "aoe_increment": 0.25,
+    "min_range": 5,
+    "max_range": 20,
+    "range_increment": 1
+  },
+  {
+    "id": "magus_mana_bolt",
+    "type": "SPELL",
+    "name": "Mana Bolt",
+    "description": "A bolt of magical power that only damages your foes.",
+    "valid_targets": [ "hostile" ],
+    "effect": "projectile_attack",
+    "spell_class": "MAGUS",
+    "energy_source": "MANA",
+    "base_casting_time": 100,
+    "base_energy_cost": 150,
+    "difficulty": 5,
+    "max_level": 20,
+    "min_damage": 20,
+    "max_damage": 120,
+    "damage_increment": 5,
+    "damage_type": "none",
+    "min_range": 5,
+    "max_range": 30,
+    "range_increment": 1.5
+  },
+  {
+    "id": "magus_haste",
+    "type": "SPELL",
+    "name": "Haste",
+    "description": "This spell gives you an enormous boost of speed lasting a short period of time.",
+    "valid_targets": [ "self" ],
+    "effect": "target_attack",
+    "effect_str": "haste",
+    "effected_body_parts": [ "TORSO" ],
```

Comment:
you could remove this line and it will affect the player in general, instead of being applied to the torso, but that's just a stylistic thing.

Reply:
Oh, I must have misunderstood you before then. I'll remove it.